src/textacy/extract/triples.py

```diff
@@ -27,9 +27,10 @@
     nsubjpass,
     obj,
     pobj,
-    xcomp,
+    xcomp
```

Comment:
this comma was here for a reason -- `black` put it there automatically :)

```suggestion
    xcomp,
```